db/redshift.go

```diff
@@ -0,0 +1,126 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/HomesNZ/go-common/env"
+
+	// SQL driver
+	_ "github.com/lib/pq"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/cenkalti/backoff"
+)
+
+var (
+
+	// connRedshift is the current redshift connection
+	connRedshift *sql.DB
+
+	// onceRedshift prevents InitConnection from being called more than once in Conn
+	onceRedshift = sync.Once{}
+)
+
+// InitConnectionRedshift creates a new new connection to the database and verifies that it succeeds.
+func InitConnectionRedshift() {
+	db := RS{}
```

Comment:
Should be something like 'rs', looks like you're using the package name as a variable